### ISSUE TITLE
Improvements for /inspect EMS Command

### ISSUE THREAD

**Issue by CloudTheWolf** [InZidiuZ/op-framework-issue-tracker#2248]
**Assurance:**
Have you read through the rules from the `README.md` file in the root folder of this GitHub repository?
Yes
**Summary:**

 Change `/inspect` so that it displays in the chat instead of how it shows Doctor duty.

Eg  you do /inspect and in the chat it says

```
Patient is not bleeding
Patient has minor injury on Left Upper Arm
```

Or 

 
```
Patient has Heavy Bleeding from the chest
Patient has Light Bleeding from the head
Patient has Upper Chest Injury
Patient has Upper Spine Injury
```

Or
```
Patient is not bleeding
Patient has no injuries
```

Instead of just "Multiple Injuries" or "No Injuries" etc

This would be similar to the `/check_vehicle_upgrades` command for mechanics 

**Reason:**
This would mean EMS / First Responders could more efficiently diagnose patients on scene.

**Comment by twooot** [comment]
The intention behind the /inspect command was to give EMS the information of: is the patient injured or not and how serverely are they injured? My idea was you'D check how severely someone is injured and then bring them to the closest hospital and do a closer inspection there using the doctor duty

**Comment by TheOneLizard** [comment]
> The intention behind the /inspect command was to give EMS the information of: is the patient injured or not and how serverely are they injured? My idea was you'D check how severely someone is injured and then bring them to the closest hospital and do a closer inspection there using the doctor duty

To get in on this, as a paramedic you still want to stabilise the patient on scene before you get them transported, hence knowing all their injuries is very much necessary. Additionally, if you are alone on duty and with doctor/bedside duty being used less, it will rarely be the case that we treat the patient further after checking them in if it isn't an ICU case.

**Comment by twooot** [comment]
Closed in favor of #2399
